src/Components/AdvancedSearch/Internal/AdvancedSearchResultDetailsList/AdvancedSearchResultDetailsList.tsx

```diff
@@ -199,11 +198,37 @@ const AdvancedSearchResultDetailsList: React.FC<IAdvancedSearchResultDetailsList
         }
     };
 
+    const getAvailableProperties = () => {
+        let properties: string[] = [];
+        twins.forEach((twin) => {
+            properties = Object.keys(twin).concat(properties);
+        });
+        properties.sort((a, b) => (a.toLowerCase() > b.toLowerCase() ? 1 : -1));
+        return new Set(properties);
+    };
+    const updateColumns = (columnToAdd: string) => {
+        setSelectedColumnNames((currentValue) => {
+            return currentValue.concat(columnToAdd);
+        });
+    };
+    const deleteColumn = (columnToRemoveKey: string) => {
+        setSelectedColumnNames((currentValue) =>
+            currentValue.filter((col) => col !== columnToRemoveKey)
+        );
+    };
     return (
         <section className={classNames.root}>
-            <h3 className={classNames.listHeader}>
-                {t('advancedSearch.results', { twinCount })}
-            </h3>
+            <div className={classNames.listHeaderAndIcon}>
+                <h3 className={classNames.listHeader}>
+                    {t('advancedSearch.results', { twinCount })}
+                </h3>
+                <ColumnPicker
+                    searchedProperties={searchedProperties}
```

Comment:
ColumnPicker is a styled component, please add the className as a subcomponent style.